superset/assets/src/explore/exploreUtils.js

```diff
@@ -182,3 +183,24 @@ export function exportChart(formData, endpointType) {
   exploreForm.submit();
   document.body.removeChild(exploreForm);
 }
+
+export function exportChartPNG(title, index, divStr) {
+  const _canvas = document.querySelectorAll(divStr)[index];
+  const w = parseInt(window.getComputedStyle(_canvas).width);
+  const h = parseInt(window.getComputedStyle(_canvas).height);
+
+  const canvas = document.createElement('canvas');
```

Comment:
the values which arepresent inside scroll are not screenshot

Reply:
Sorry, there is an error in my submission

Reply:
i am trying to take scrollheight for it but nothing is useful

Reply:
There are one mistakes above and it will be ok after modification

![屏幕快照 2019-03-15 下午5 47 49](https://user-images.githubusercontent.com/8774296/54422932-82419980-474a-11e9-93ab-a2227e96bf85.png)